src/bootstrap/lib.rs

```diff
@@ -280,6 +280,14 @@ impl Crate {
     }
 }
 
+// FIXME: add comment
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub enum DependencyType {
+    Host,
+    SelfContained,
```

Comment:
```suggestion
    TargetSelfContained,
```
It should be more clear like this.
(I'd also make it the last variant because, well, it's the least important.)